dll/win32/shell32/shellrecyclebin/recyclebin_generic.cpp

```diff
@@ -97,7 +97,15 @@ STDMETHODIMP RecycleBinGeneric::DeleteFile(LPCWSTR szFileName)
 
     /* Get associated volume path */
     WCHAR szVolume[MAX_PATH];
+#ifndef __REACTOS__
```

Comment:
Could you test whether, removing this ifndef REACTOS (+ the swprintf below) altogether would still work? I think so, because we have GetVolumePathNameW implemented (and other places in the recycler code already use it, as I can see in your other PRs), so I think it should be possible to unconditionally use it there as well.

Reply:
`RecycleBinGeneric::DeleteFile` is not called from anywhere.

Reply:
How do we implement file deletion, or emptying the recycle bin?